### PR TITLE
Update cross-origin isolation guide message

### PR DESCRIPTION
**Pull request by agektmr**
Update cross-origin isolation guide message

cc: @lutzvahl

### CODE REVIEW

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 1]
```diff
@@ -18,11 +18,78 @@ tags:
...
+[SharedArrayBuffer](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer),
 [`performance.measureUserAgentSpecificMemory()`](/monitor-total-page-memory-usage/),
 or the JS Self-Profiling API.
 
-## Do an impact analysis
+{% Aside 'objective' %}
+If you landed this page by a SharedArrayBuffer deprecation message, it's likely
+either your website or one of the resources you emebed to your website is using
+SharedArrayBuffer. When Chrome hits 91, the functionality that has been
+available through SharedArrayBuffer will no longer work. take one of the
```
Comment (lutzvahl) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:30]: It still works, but only in COI context.

Comment (lutzvahl) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:30]: [...] take -> Take

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 2]
```diff
@@ -18,11 +18,78 @@ tags:
...
+following actions to fix the issue in advance.
+
+* Turn on the cross-origin isolation to keep using SharedArrayBuffer.
+* Notify the third-party that actually uses SharedArrayBuffer to take action.
```
Comment (lutzvahl) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:34]: Notify the third-party provider/vendor(?)

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 3]
```diff
@@ -18,11 +18,78 @@ tags:
...
+{% endAside %}
+
+## Determine where in your website a SharedArrayBuffer is used
+
+If you are not sure where in your site a SharedArrayBuffer is used, there are
+two ways:
+
+* Using Chrome DevTools
```
Comment (lutzvahl) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:42]: Shall we mark Chrome DevTools as 'Basic' and Reporting API as 'Advanced'?

**Review by lutzvahl — COMMENTED** [review]
Thanks for putting this together!

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 4]
```diff
@@ -18,11 +18,79 @@ tags:
...
+If you landed this page by a SharedArrayBuffer deprecation message, it's likely
+either your website or one of the resources you emebed to your website is using
+SharedArrayBuffer. When Chrome hits 91, the functionality that has been
+available through SharedArrayBuffer will no longer work without cross-origin
+isolation. Take one of the following actions to fix the issue in advance.
+
+* Turn on the cross-origin isolation to keep using SharedArrayBuffer.
+* Notify the third-party vendor/provider that actually uses SharedArrayBuffer to
+  take action.
+{% endAside %}
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:36]:
```suggestion
{% Banner 'info', 'body' %}
Starting in Chrome 91, functionalities that use `SharedArrayBuffer` will no longer 
work without cross-origin isolation. If you landed on this page due to a 
`SharedArrayBuffer` deprecation message, it's likely either your website or one of 
the resources embedded on it is using `SharedArrayBuffer`. To ensure nothing breaks 
on your website due to deprecation, follow the steps in this post.
{% endBanner %}


{% Aside 'objective' %}
* Turn on cross-origin isolation to keep using `SharedArrayBuffer`.
* If you rely on third-party code that uses `SharedArrayBuffer`, notify the third-party 
  provider to take action.
{% endAside %}
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 5]
```diff
@@ -18,11 +18,79 @@ tags:
...
+## Determine where in your website a SharedArrayBuffer is used
+
+If you are not sure where in your site a SharedArrayBuffer is used, there are
+two ways:
+
+* [Basic] Using Chrome DevTools
+* [Advanced] Using Deprecation Reporting
+
+Skip to [Do an impact analysis](:analysis) if you already know where you are
+using SharedArrayBuffer.
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:47]:
```suggestion
## Determine where in your website `SharedArrayBuffer` is used

If you are not sure where in your site a `SharedArrayBuffer` is used, there are
two ways find out:

* Using Chrome DevTools
* (Advanced) Using Deprecation Reporting

If you already know where you are using `SharedArrayBuffer`, skip to 
[Do an impact analysis](:analysis).
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 6]
```diff
@@ -18,11 +18,79 @@ tags:
...
+      ```http
+      Report-To: {"group":"default","max_age":86400,"endpoints":[{"url":"THE_DEPRECATION_ENDPOINT_URL"}]}
+      ```
+3. Once the header starts to propagate, the endpoint you registered to should
+   start collecting deprecation reports.
+
+See an example implementation here:
+[https://first-party-test.glitch.me](https://first-party-test.glitch.me)
+
+## Do an impact analysis {: #analysis}
```
Comment (sebastianbenz) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:93]: I've found this headline a bit confusing. Suggestion: 'Analyse the impact of cross-origin isolation'

**Review by sebastianbenz — COMMENTED** [review]
The intro is much better. Few more comments:

* The following is in the "Mitigate the impact of cross-origin isolation", but is a duplication og the same recommendation from the impact analysis section:
  _On cross-origin resources loaded into iframes, set the Cross-Origin-Embedder-Policy-Report-Only: require-corp header to do an impact analysis._
* The impact analysis would benefit from a one sentence explanation of what Cross-Origin-Opener-Policy-Report-Only and Cross-Origin-Embedder-Policy-Report-Only do.

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 7]
```diff
@@ -18,11 +18,79 @@ tags:
...
+   the page you suspect it might be using a SharedArrayBuffer.
+2. Select the **Console** panel.
+3. The following message will show up if SharedArrayBuffer is used:
+      ```text
+      [Deprecation] SharedArrayBuffer will require cross-origin isolation as of M91, around May 2021. See https://developer.chrome.com/blog/enabling-shared-array-buffer/ for more details. common-bundle.js:535
+      ```
+4. The filename and the line number indicates where the SharedArrayBuffer comes
+   from. If it's a third-party library, please contact the developer to fix the
+   issue. If it's implemented as part of your website, follow the guide below to
+   enable cross-origin isolation.
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:65]:
```suggestion
### Using Chrome DevTools

[Chrome DevTools](https://developers.google.com/web/tools/chrome-devtools/open)
allows developers to inspect websites. 

1. [Open the Chrome
   DevTools](https://developers.google.com/web/tools/chrome-devtools/open) on
   the page you suspect might be using `SharedArrayBuffer`.
2. Select the **Console** panel.
3. If the page is using `SharedArrayBuffer`, the following message will show up:
      ```text
      [Deprecation] SharedArrayBuffer will require cross-origin isolation as of M91, around May 2021. See https://developer.chrome.com/blog/enabling-shared-array-buffer/ for more details. common-bundle.js:535
      ```
4. The filename and the line number at the end of the message (for example, `common-bundle.js:535`) 
   indicate where the `SharedArrayBuffer` is coming from. If it's a third-party library, 
   contact the developer to fix the issue. If it's implemented as part of your website, follow 
   the guide below to enable cross-origin isolation.
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 8]
```diff
@@ -18,11 +18,79 @@ tags:
...
+
+<figure class="w-figure">
+{% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/GOgkyjAabePTc8AG22F7.png", alt="DevToools Console warning when SharedArrayBuffer is used without cross-origin isolation", width="800", height="163" %}
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:68]:
```suggestion
{% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/GOgkyjAabePTc8AG22F7.png", alt="DevToools Console warning when SharedArrayBuffer is used without cross-origin isolation", width="800", height="163", class="w-screenshot" %}
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 9]
```diff
@@ -18,11 +18,79 @@ tags:
...
+   <figcaption>
+      DevToools Console warning when SharedArrayBuffer is used without cross-origin isolation
+   </figcaption>
+</figure>
+
+### [Advanced] Using Deprecation Reporting
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:74]:
```suggestion
### (Advanced) Using Deprecation Reporting
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 10]
```diff
@@ -18,11 +18,79 @@ tags:
...
+      ```
+4. The filename and the line number indicates where the SharedArrayBuffer comes
+   from. If it's a third-party library, please contact the developer to fix the
+   issue. If it's implemented as part of your website, follow the guide below to
+   enable cross-origin isolation.
+
+<figure class="w-figure">
+{% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/GOgkyjAabePTc8AG22F7.png", alt="DevToools Console warning when SharedArrayBuffer is used without cross-origin isolation", width="800", height="163" %}
+   <figcaption>
+      DevToools Console warning when SharedArrayBuffer is used without cross-origin isolation
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:70]:
```suggestion
      DevToools Console warning when `SharedArrayBuffer` is used without cross-origin isolation.
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md (hunk 6 above)
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:93]:
```suggestion
## Analyze the impact of cross-origin isolation  {: #analysis}
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 11]
```diff
@@ -18,11 +18,79 @@ tags:
...
+2. Using the URL, set the following HTTP header to pages that are potentially
+   serving SharedArrayBuffer usage.
+      ```http
+      Report-To: {"group":"default","max_age":86400,"endpoints":[{"url":"THE_DEPRECATION_ENDPOINT_URL"}]}
+      ```
+3. Once the header starts to propagate, the endpoint you registered to should
+   start collecting deprecation reports.
+
+See an example implementation here:
+[https://first-party-test.glitch.me](https://first-party-test.glitch.me)
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:91]:
```suggestion
Some browsers have [a reporting functionality of deprecating
APIs](https://wicg.github.io/deprecation-reporting/) to a specified endpoint.

1. [Set up a deprecation report server and get the reporting
   URL](/coop-coep/#set-up-reporting-endpoint). You can achieve this by either
   using a public service or building one yourself.
2. Using the URL, set the following HTTP header to pages that are potentially
   serving `SharedArrayBuffer`.
      ```http
      Report-To: {"group":"default","max_age":86400,"endpoints":[{"url":"THE_DEPRECATION_ENDPOINT_URL"}]}
      ```
3. Once the header starts to propagate, the endpoint you registered to should
   start collecting deprecation reports.

See an example implementation here:
[https://first-party-test.glitch.me](https://first-party-test.glitch.me).
```

**Comment by mihajlija** [comment]
@sebastianbenz I've applied your suggestions, thanks! Because this is meant to be very focused on "how to" achieve cross-origin isolation, I added links to explanations instead of inlining them.

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 12]
```diff
@@ -18,17 +18,88 @@ tags:
 ---
 
 This guide shows you how to enable cross-origin isolation. Cross-origin
-isolation is required if you want to use SharedArrayBuffer,
+isolation is required if you want to use
+[SharedArrayBuffer](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer),
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:22]:
```suggestion
[`SharedArrayBuffer`](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/SharedArrayBuffer),
```

File: src/site/content/en/secure/cross-origin-isolation-guide/index.md [hunk 13]
```diff
@@ -18,17 +18,88 @@ tags:
...
+      ```
+4. The filename and the line number at the end of the message (for example, `common-bundle.js:535`) 
+   indicate where the `SharedArrayBuffer` is coming from. If it's a third-party library, 
+   contact the developer to fix the issue. If it's implemented as part of your website, follow 
+   the guide below to enable cross-origin isolation.
+
+<figure class="w-figure">
+{% Img src="image/YLflGBAPWecgtKJLqCJHSzHqe2J2/GOgkyjAabePTc8AG22F7.png", alt="DevToools Console warning when SharedArrayBuffer is used without cross-origin isolation", width="800", height="163", class="w-screenshot" %}
+   <figcaption>
+      DevToools Console warning when `SharedArrayBuffer` is used without cross-origin isolation.
```
Comment (mihajlija) [inline src/site/content/en/secure/cross-origin-isolation-guide/index.md:73]:
```suggestion
      DevToools Console warning when SharedArrayBuffer is used without cross-origin isolation.
```

**Pull request merged**